v20.2/create-view.md

```diff
@@ -21,6 +21,7 @@ The user must have the `CREATE` [privilege](authorization.html#assign-privileges
 Parameter | Description
 ----------|------------
 `IF NOT EXISTS` |  Create a new view only if a view of the same name does not already exist. If one does exist, do not return an error.<br><br>Note that `IF NOT EXISTS` checks the view name only. It does not check if an existing view has the same columns as the new view.
+`OR REPLACE`  |  <span class="version-tag">New in v20.2:</span> If a view of the same name already exists, replace that view.<br><br>Note that `OR REPLACE` checks the view name only. It does not check if an existing view has the same columns as the new view.
```

Comment:
This is not quite correct. When we do replace view, we ensure that the new view has at least all of the same columns in the existing view as a prefix.

For example, if the old view had columns (a, b) the new view must have columns (a, b, insert, new, columns, here).

Reply:
TFTR! I just updated the parameter description following your feedback.